Data/Repositories/InterviewRepository.cs

```diff
@@ -12,5 +12,23 @@ public InterviewRepository(DataContext context) : base(context)
         {
             _dataContext = context;
         }
+
+        public Interview GetInterview(int id)
+        {
+            Interview interview = _dataContext.Interviews.Where(i => i.Id == id)
+                .Select(x => new Interview
```

Comment:
You can simply Include Candidate a get Interview+Candidate only by FirstOrDefault, without Select